### ISSUE TITLE
About the experimesnt of 'Effect of Number of Positives'

### ISSUE THREAD

**Issue by Chen-Song** [HobbitLong/SupContrast#33]
How did you do this experiment? That is, how to control the number of positive samples? My understanding is that when the number of positive samples is 5, that is, when the batchsize is set to 5000, because 5000/1000=5, the result is 78.8. But the batch size set in the paper is 8192, and the performance obtained is 78.8.

**Comment by HobbitLong** [comment]
I did not perform that experiments so not very sure about that detail, but that table will be further updated in a later version.

**Comment by Nusselder9** [comment]
Hi Song. I think `My understanding is that when the number of positive samples is 5, that is, when the batchsize is set to 5000, because 5000/1000=5` is not correct. Even if you set batchsize as 5, it may contain 5 positives. Because `torch.utils.data.DataLoader` sample images randomly without label informations.
